### PR TITLE
Merging Admin Updates to Main 2020-Redesign Branch

### PR DESCRIPTION
**Pull request by amanifarooque**
Many updates to the site structure, page layout, and styling are included in this merge.

### CODE REVIEW

File: _includes/logo.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
 <div class="usa-logo">
-  <em class="usa-logo__text"><a href="{{site.baseurl}}/" title="Home" aria-label="Home">{% asset searchgovlogo.png class="logo" alt="search.gov" width="175" height="30" %}</a></em>
+  <em class="usa-logo__text">
+    <a href="{{site.baseurl}}/" title="Home" aria-label="Home">
```
Comment (dawnpm) [inline _includes/logo.html:3]: If this link isn't working, we might need to add space characters inside of the curly brackets

Comment (amanifarooque) [reply _includes/logo.html:3]: @dawnpm It seems like the link is working in the preview link - starting on a different [page](https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/afarooque/2020-redesign-netlifycms/about/customers.html) the logo links to the main homepage. Any requested changes here?

File: about/tos.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 ---
```
Comment (dawnpm) [inline about/tos.md:1]: We need to move this one level down to inside of /about/policy/

File: admin-center/activate/index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+sidenav: admin_center
+---
+* [Code snippets]({{ site.baseurl }}/manual/code.html)
+* [Search Results API instructions and access key](https://open.gsa.gov/api/ searchgov-results/)\*
+* [Type-ahead API instructions](https://open.gsa.gov/api/searchgov-suggestions/)\*
+* [i14y API instructions]({{ site.baseurl }}/developer/i14y.html)\*
+
+\* These APIs are available for use on official government websites only. You must be a Search.gov customer. [Sign in](https://search.usa.gov/sites) is required. Keys and instructions for each can be found under [Admin Center](https://search.usa.gov/sites) > YourSite > Activate Search.
+
+{% include under-construction.html %}
```
Comment (dawnpm) [inline admin-center/activate/index.md:13]: why doesn't this page show my changes? hope they didn't get lost.

File: admin-center/analytics/third-party.md [hunk 4]
```diff
@@ -9,6 +9,11 @@ breadcrumbs:
     url: /
   - label: Manual
     url: /manual/
```
Comment (dawnpm) [inline admin-center/analytics/third-party.md:11]: Need to update all the Help Manual breadcrumbs

Comment (dawnpm) [reply admin-center/analytics/third-party.md:11]: Or just remove this one?

File: admin-center/display/govbox-jobs.md [hunk 5]
```diff
@@ -21,4 +28,4 @@ Searchers will also see a direct link to [USAJobs.gov](https://www.usajobs.gov)
 
 ---
 
- **Did you know?** Are you a federal agency that doesn't post your jobs on USAJobs? We want your jobs data. Email us at <search@support.digitalgov.gov> to let us know where you publish your job openings so that we can include them in the Job Openings module.
+If you would like to show [job openings]({{ site.baseurl }}/manual/govbox-jobs.html) in your results, but do not see the toggle to turn them on, [email us](mailto:search@support.digitalgov.gov) for assistance.
```
Comment (dawnpm) [inline admin-center/display/govbox-jobs.md:31]: just realized that the link in this line points back to this very page. Please remove, thanks!

File: admin-center/display/govbox-federal-register.md [hunk 6]
```diff
@@ -1,14 +1,23 @@
...
 
 If you [opt to show the Federal Register]({{ site.baseurl }}/manual/display-overview.html) module, searchers on your site can readily find the notices and rules that your agency has published in the *Federal Register*, as shown in the sample below for a search on [*autoimmune diseases* on NIH.gov](https://search.nih.gov/search?affiliate=nih&query=autoimmune+diseases).
 
-[![Federal Register Module on NIH.gov](https://d3qcdigd1fhos0.cloudfront.net/blog/img/govbox-fr.png "Federal Register Module on NIH.gov")](https://search.nih.gov/search?affiliate=nih&query=autoimmune+diseases)
\ No newline at end of file
+[![Federal Register Module on NIH.gov](https://d3qcdigd1fhos0.cloudfront.net/blog/img/govbox-fr.png "Federal Register Module on NIH.gov")](https://search.nih.gov/search?affiliate=nih&query=autoimmune+diseases)
+
+If you would like to show [*Federal Register*]({{ site.baseurl }}/manual/govbox-federal-register.html)documents in your results, but do not see the toggle to turn them on, [email us](mailto:search@support.digitalgov.gov) for assistance.
```
Comment (dawnpm) [inline admin-center/display/govbox-federal-register.md:23]: same as for the jobs page, realized that the link here points back to this same page. please remove.

File: admin-center/display/govbox-health.md [hunk 7]
```diff
@@ -1,12 +1,21 @@
...
 
+Find it in the Admin Center: [Search.gov Home]({{ site.baseurl }}/index.html) > [Admin Center](https://search.usa.gov/sites/) > YourSite > Manage Display > Display Overview
+
 Searches for health-related information are very common. We created the Health Topics module to help searchers find high-quality, authoritative health information.
 
 This module offers direct access to [MedlinePlus Health Topics](https://www.nlm.nih.gov/medlineplus/healthtopics.html) and open clinical studies and trials on [ClinicalTrials.gov](https://clinicaltrials.gov/). To provide timely information across all health topics, we pull the data from the [MedlinePlus XML files](https://www.nlm.nih.gov/medlineplus/xml.html), which cover both the English and Spanish health topics.
 
 If you [opt to show the Health Topics]({{ site.baseurl }}/manual/display-overview.html) module, searchers looking for health topics on your site can readily find high-quality health information.
+
+If you would like to show [health topics]({{ site.baseurl }}/manual/govbox-health.html) in your results, but do not see the toggle to turn them on, [email us](mailto:search@support.digitalgov.gov) for assistance.
```
Comment (dawnpm) [inline admin-center/display/govbox-health.md:21]: same as for jobs and federal register, I realized this link points back to this very same page. Please remove.

File: admin-center/display/index.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Manage Display
+sidenav: admin_center
```
Comment (dawnpm) [inline admin-center/display/index.md:4]: Should we add more front matter?

File: admin-center/display/system-alert.md [hunk 9]
```diff
@@ -1,8 +1,13 @@
...
 category: manual
 tags: help-manual manage-display alert
+date: February 7, 2018
+last_modified_at: February 7, 2018
+sidenav: admin_center
+redirect_from:
+    - /manual/system-alert.html
 ---
 
 [Search.gov Home]({{ site.baseurl }}/index.html) > [Admin Center](https://search.usa.gov/sites/) > YourSite > Display > Search Page Alert
```
Comment (dawnpm) [inline admin-center/display/system-alert.md:13]: I missed this one. Please prepend this line with `Find it in the Admin Center: `

File: get-started/account.md [hunk 10]
```diff
@@ -1,14 +1,19 @@
 ---
-layout: post
+layout: page
 title: How to Create and Edit Your User Account
 category: manual
 tags: help-manual my-account
 crumbname: Manage User Account
+date: June 23, 2020
+last_modified_at: June 23, 2020
 breadcrumbs:
```
Comment (dawnpm) [inline get-started/account.md:9]: We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs

File: get-started/go-live.md [hunk 11]
```diff
@@ -1,15 +1,20 @@
...
-layout: post
+layout: page
 title: Going Live with Search.gov
 crumbname: Go-Live
 category: manual
-redirect_from: "/blog/go-live.html"
 tags: help-manual
+date: September 4, 2019
+last_modified_at: September 4, 2019
 breadcrumbs:
```
Comment (dawnpm) [inline get-started/go-live.md:9]: We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs

File: get-started/go-live.md [hunk 12]
```diff
@@ -35,11 +40,21 @@ Most agencies add these two [snippets of code]({{ site.baseurl }}/manual/code.ht
 
 <i class="icon-check"></i> **7. Does your search box point to Search.gov?** Update your search box form code point to our service so that, when users execute a search, they'll see your Search.gov-hosted results page.
 
-	<form accept-charset="UTF-8" action="https://search.usa.gov/search" id="search_form" method="get"><div style="margin:0;padding:0;display:inline"><input name="utf8" type="hidden" value="&#x2713;" /></div>
-		 <input id="affiliate" name="affiliate" type="hidden" value="YourSiteHandle" />
-	 	<label for="query">Enter Search Term(s):</label>
-	 	<input autocomplete="off" class="usagov-search-autocomplete" id="query" name="query" type="text" />
-	 	<input name="commit" type="submit" value="Search" />
+	<form accept-charset="UTF-8"
```
Comment (dawnpm) [inline get-started/go-live.md:43]: Need to make sure this displays as a block of code, and not as an actual search box.

File: get-started/site-launch-guide.md [hunk 13]
```diff
@@ -1,14 +1,19 @@
 ---
-layout: post
+layout: page
 title: Search Site Launch Guide
 crumbname: Site Launch Guide
 category: manual
 tags: go-live site-launch indexing
+date: April 22, 2019
+last_modified_at: February 3, 2021
 breadcrumbs:
```
Comment (dawnpm) [inline get-started/site-launch-guide.md:9]: We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs

File: get-started/training.md [hunk 14]
```diff
@@ -1,14 +1,19 @@
 ---
-layout: post
+layout: page
 title: Search.gov Training
 crumbname: Training
 category: manual
 tags: help-manual training videos
+date: June 20, 2019
+last_modified_at: June 20, 2019
 breadcrumbs:
```
Comment (dawnpm) [inline get-started/training.md:9]: We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs

File: indexing/how-search-engines-index-content-better-discoverability.md [hunk 15]
```diff
@@ -1,17 +1,22 @@
 ---
-layout: post
+layout: page
 title: How to get search engines to index the right content for better discoverability
 crumbname: Better Discoverability
 category: manual
```
Comment (dawnpm) [inline indexing/how-search-engines-index-content-better-discoverability.md:5]: Let's do a bulk update to change `category: manual` into `category: admin-center`. OR update the term AND convert to a tag.

File: admin-center/activate/index.md (hunk 3 above)
Comment (amanifarooque) [reply admin-center/activate/index.md:13]: Going back through the commits I don't think your changes were ever committed, they might be lost :/

File: admin-center/dashboard/add-site.md [hunk 16]
```diff
@@ -1,12 +1,16 @@
...
-layout: post
+layout: page
 title: How to Add a New Site
 category: manual
 tags: "help-manual settings add-site"
+date: March 2, 2017
+last_modified_at: March 2, 2017
 published: true
+sidenav: admin_center
+redirect_from: "/manual/add-site"
```
Comment (dawnpm) [inline admin-center/dashboard/add-site.md:10]: This is the only redirect that doesn't have the url on an indented line following.

Comment (amanifarooque) [reply admin-center/dashboard/add-site.md:10]: Both seem to work, but will update!

File: indexing/how-search-engines-index-content-better-discoverability.md (hunk 15 above)
Comment (amanifarooque) [reply indexing/how-search-engines-index-content-better-discoverability.md:5]: Bulk update seems easier for now, are we linking out or using categories currently?

File: get-started/training.md (hunk 14 above)
Comment (amanifarooque) [reply get-started/training.md:9]: Going to remove these breadcrumbs since we have the sidenav now.

File: admin-center/activate/code.md [hunk 17]
```diff
@@ -1,8 +1,13 @@
...
-layout: post
+layout: page
 title: How to Activate Your Search by Adding Our Code Snippets to Your Website
 category: manual
 tags: help-manual activate-search code-snippets
+date: October 13, 2020
+last_modified_at: October 13, 2020
+sidenav: admin_center
+redirect_from:
+    - /manual/code.html
```
Comment (dawnpm) [inline admin-center/activate/code.md:10]: 4 space indentation, should be 2.

File: admin-center/activate/typeahead-api.md [hunk 18]
```diff
@@ -1,8 +1,13 @@
...
-layout: post
+layout: page
 title: Displaying Type-ahead Suggestions on Your Agency's Website
 category: manual
 tags: api activate-search autocomplete
+date: October 26, 2017
+last_modified_at: October 26, 2017
+sidenav: admin_center
+redirect_from:
+    - /manual/typeahead-api.html
```
Comment (dawnpm) [inline admin-center/activate/typeahead-api.md:10]: 4 space indentation, should be 2

File: admin-center/analytics/analytics-alerts.md [hunk 19]
```diff
@@ -1,11 +1,16 @@
...
-layout: post
+layout: page
 title: Setting Up Analytics Alerts
 category: manual
 tags: help-manual analytics
+date: March 23, 2017
+last_modified_at: March 23, 2017
+sidenav: admin_center
+redirect_from:
+    - /manual/analytics-alerts.html
```
Comment (dawnpm) [inline admin-center/analytics/analytics-alerts.md:10]: 4 space indentation, should be 2

File: admin-center/analytics/clicks.md [hunk 20]
```diff
@@ -1,17 +1,23 @@
...
-layout: post
+layout: page
 title: Analyzing Your Site's Clicks
 category: manual
 tags: help-manual analytics clicks
+date: February 28, 2018
+last_modified_at: February 28, 2018
+sidenav: admin_center
+redirect_from:
+    - /manual/clicks.html
```
Comment (dawnpm) [inline admin-center/analytics/clicks.md:10]: 4 space indentation, should be 2. Probably good to do a global update for this.

File: about/policy/govt-urls.md [hunk 21]
```diff
@@ -1,20 +1,24 @@
...
-layout: post
+layout: page
 title: Government-Managed Domains Outside the .Gov and .Mil Top Level Domains
 category: developer
 tags: seo policy government-urls
+sidenav: about_us
+date: March 2, 2019
+last_modified_at: March 2, 2019
+redirect_from:
+  - /developer/govt-urls.html
```
Comment (dawnpm) [inline about/policy/govt-urls.md:10]: make 4 spaces

File: admin-center/content/domains.md [hunk 22]
```diff
@@ -1,11 +1,16 @@
...
-layout: post
+layout: page
 title: How to Edit Your Domains
 category: manual
 tags: help-manual manage-content domains
+date: April 18, 2017
+last_modified_at: April 18, 2017
+sidenav: admin_center
+redirect_from:
+  - /manual/domains.html
```
Comment (dawnpm) [inline admin-center/content/domains.md:10]: make 4 spaces

File: admin-center/index.md [hunk 23]
```diff
@@ -8,76 +8,81 @@ last-updated: "April 10, 2019"
 breadcrumbs:
   - label: Home
     url: /
+sidenav: admin_center
+redirect_from:
+  - /manual/
```
Comment (dawnpm) [inline admin-center/index.md:13]: make 4 spaces

File: _config.yml [hunk 24]
```diff
@@ -127,32 +140,123 @@ primary_navigation:
...
+        url: "/get-started/site-launch-guide.html#add-features"
+
 admin_center:
-  - name: Admin Center
+  - name: Manage Your Search Configuration
     url: /admin-center/
-    children:
-      - name: Color Codes
-        url: /admin-center/color-codes.html
+    children:
```
Comment (dawnpm) [inline _config.yml:173]: Add quotation marks around all these nav items

File: _config.yml [hunk 25]
```diff
@@ -127,32 +140,123 @@ primary_navigation:
...
+          - name: What Search.gov indexes from your website
+            url: "/indexing/what-searchgov-indexes.html"
+          - name: How Search.gov ranks your search results
+            url: "/indexing/ranking-factors.html"
+          - name: How A Page on a Sitemap Becomes a Search
+            url: "/indexing/sitemaps-to-search-sites.html"
+          - name: A step-by-step indexing workflow
+            url: "/get-started/site-launch-guide.html#select-index"
+      - name: Domain Level SEO Supports
+        url:
```
Comment (dawnpm) [inline _config.yml:237]: remove empty url line

File: _config.yml [hunk 26]
```diff
@@ -127,32 +140,123 @@ primary_navigation:
...
+        children:
+          - name: All about XML sitemaps
+            url: "/indexing/sitemaps.html"
+          - name: Robots.txt files
+            url: "/indexing/robotstxt.html"
+      - name: Page Level SEO Supports
+        url:
```
Comment (dawnpm) [inline _config.yml:244]: Remove empty url line

File: admin/config.yml [hunk 27]
```diff
@@ -4,17 +4,17 @@ backend:
   base_url: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/2020-redesign/
   auth_endpoint: external/auth/github
   preview_context: federalist/build
-  branch: 2020-redesign
+  branch: afarooque/2020-redesign-netlifycms
```
Comment (dawnpm) [inline admin/config.yml:7]: Could the mismatch between this and line 4 be why the subnav menus aren't working?

File: indexing/how-search-engines-index-content-better-discoverability.md [hunk 28]
```diff
@@ -1,17 +1,22 @@
...
-redirect_from: "/blog/how-search-engines-index-content-better-discoverability.html"
+redirect_from: 
+  - "/blog/how-search-engines-index-content-better-discoverability.html"
+  - /manual/how-search-engines-index-content-better-discoverability.html
 tags: indexing seo
+date: February 27, 2020
+last_modified_at: February 27, 2020
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/how-search-engines-index-content-better-discoverability.md:12]: update or remove

File: indexing/metadata.md [hunk 29]
```diff
@@ -1,9 +1,11 @@
...
+layout: page
 title: Metadata and tags you should include in your website
 crumbname: Metadata
 category: manual
-redirect_from: "/blog/metadata.html"
+redirect_from: 
+  - /blog/metadata.html
+  - /manual/metadata.html
 tags: indexing seo
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/metadata.md:10]: update or remove

File: indexing/ranking-factors.md [hunk 30]
```diff
@@ -1,16 +1,21 @@
 ---
-layout: post
+layout: page
 title: How Search.gov Ranks Your Search Results
 crumbname: Ranking Factors
 category: manual
 tags: indexing seo
+date: July 9, 2019
+last_modified_at: July 9, 2019
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/ranking-factors.md:9]: update or remove

File: indexing/redesign.md [hunk 31]
```diff
@@ -1,17 +1,23 @@
...
 crumbname: Redesign Guide
 category: manual
-redirect_from: "/blog/redesign.html"
+redirect_from: 
+  - /blog/redesign.html
+  - /manual/redesign.html
 tags: seo site-launch indexing
+date: August 30, 2019
+last_modified_at: August 30, 2019
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/redesign.md:12]: update or remove

File: indexing/robotstxt.md [hunk 32]
```diff
@@ -1,17 +1,22 @@
...
 crumbname: Robots.txt
 category: manual
-redirect_from: "/blog/robotstxt.html"
+redirect_from: 
+  - /blog/robotstxt.html
+  - /manual/robotstxt.html
 tags: how-to manage-content indexing
+date: January 17, 2020
+last_modified_at: January 17, 2020
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/robotstxt.md:12]: update or remove

File: indexing/sitemaps-to-search-sites.md [hunk 33]
```diff
@@ -1,16 +1,21 @@
 ---
-layout: post
+layout: page
 title: How a Page on a Sitemap Becomes a Search Result
 crumbname: Sitemaps and Search Results
 category: manual
 tags: go-live indexing
+date: April 9, 2019
+last_modified_at: April 9, 2019
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/sitemaps-to-search-sites.md:9]: update or remove

File: indexing/sitemaps.md [hunk 34]
```diff
@@ -1,17 +1,22 @@
...
 crumbname: Sitemaps
 category: manual
-redirect_from: "/blog/sitemaps.html"
+redirect_from: 
+  - /blog/sitemaps.html
+  - /manual/sitemaps.html
 tags: sitemaps indexing
+date: January 17, 2020
+last_modified_at: January 17, 2020
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/sitemaps.md:12]: update or remove

File: indexing/what-searchgov-indexes.md [hunk 35]
```diff
@@ -1,16 +1,22 @@
 ---
-layout: post
+layout: page
 title: What Search.gov Indexes From Your Website
 crumbname: What Search.gov Indexes
 category: manual
 tags: indexing seo
+date: April 24, 2019
+last_modified_at: April 24, 2019
 breadcrumbs:
```
Comment (dawnpm) [inline indexing/what-searchgov-indexes.md:9]: update or remove

**Review by dawnpm — CHANGES_REQUESTED** [review]
Hey there - identified a few types of change that need to be done in many places.

File: admin-center/display/index.md (hunk 8 above)
Comment (amanifarooque) [reply admin-center/display/index.md:4]: Adding "date" and "last_modified_at"

File: get-started/go-live.md (hunk 12 above)
Comment (amanifarooque) [reply get-started/go-live.md:43]: Confirmed it's displaying as a code block since it's indented!

File: admin/config.yml (hunk 27 above)
Comment (amanifarooque) [reply admin/config.yml:7]: This is just controlling the branch the Netlify CMS interface commits to

**Comment by amanifarooque** [comment]
@dawnpm Changes have been addressed, let me know if this is good to merge!

File: _includes/logo.html (hunk 1 above)
Comment (dawnpm) [reply _includes/logo.html:3]: ok, not needed then.

File: admin-center/dashboard/add-site.md [hunk 36]
```diff
@@ -1,12 +1,17 @@
...
 title: How to Add a New Site
-category: manual
+category: admin-center
 tags: "help-manual settings add-site"
+date: March 2, 2017
+last_modified_at: March 2, 2017
 published: true
+sidenav: admin_center
+redirect_from:
+    - "/manual/add-site"
```
Comment (dawnpm) [inline admin-center/dashboard/add-site.md:11]: This needs a .html, right?

**Review by dawnpm — CHANGES_REQUESTED** [review]
One more tiny change, and then we're ready to push this over. I have a few other changes that I noted down that we can make in the future.

**Comment by amanifarooque** [comment]
@dawnpm I made the last update, going to merge this!

**Pull request merged**